Add development docs (#96)
mozilla/mdv2: docs/development.md

Development
===========

GLAM is maintained on GitHub at:

  https://github.com/mozilla/glam

GLAM uses Docker for local development and deployment. Please make sure to
install [Docker] and [Docker Compose] on your computer to contribute code
or documentation changes.

[Docker]: https://docs.docker.com/engine/installation/#supported-platforms
[Docker Compose]: https://docs.docker.com/compose/install/

Configuration
-------------

To set the application up, please copy the ``.env-dist`` file to one named
``.env``.

Set the ``DJANGO_SECRET_KEY`` variable using the output of the following
command after logging into the Docker container with ``make shell``:

```
python -c "import secrets; print(secrets.token_urlsafe(50))"
```

Initial Setup
-------------

When first setting up the project you will need to set up the database and
import data that the front-end is expecting.

> The following commands require you to log into the Docker container with

Reviewer: ```suggestion
First, from the project root, log into the Docker container with
```

``make shell``.

> The first step is to run the Django migrations to create the tables:

Reviewer: ```suggestion
Then run the Django migrations to create the tables:
```
